.github/workflows/mass_deployments.yml

```diff
@@ -0,0 +1,58 @@
+name: Mass Deployments
+
+on:
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  mass-deployments:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    env:
+      NETWORK: main
```

Comment:
is this script will run only on main net? if no, we should handle this line

Reply:
As the [issue](https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1282) states, it'll run only on mainnet.